sdk/attestation/azure-security-attestation/azure/security/attestation/_models.py

```diff
@@ -69,242 +82,248 @@ class PolicyCertificatesModificationResult(object):
     """
 
     def __init__(self, certificate_thumbprint, certificate_resolution):
-        #type:(str, CertificateModification)->None
+        # type: (str, CertificateModification, AttestationToken) -> None
         self.certificate_thumbprint = certificate_thumbprint
         self.certificate_resolution = certificate_resolution
 
     @classmethod
     def _from_generated(cls, generated):
+        # type: (GeneratedPolicyCertificatesModificationResult) -> AttestationPolicyCertificateResult
         if not generated:
             return cls
         return cls(generated.certificate_thumbprint, generated.certificate_resolution)
 
-class PolicyResult(object):
-    """ PolicyResult represents the result of a :meth:`azure.security.attestation.AttestationAdministrationClient.set_policy` 
-    or :meth:`azure.security.attestation.AttestationAdministrationClient.reset_policy`  API call.
 
-    The `PolicyResult` class is returned as the body of an attestation token from
+class AttestationPolicyResult(object):
+    """AttestationPolicyResult represents the result of a
+    :meth:`azure.security.attestation.AttestationAdministrationClient.set_policy`
+    or :meth:`azure.security.attestation.AttestationAdministrationClient.reset_policy`
+    API call.
+
+    The `AttestationPolicyResult` class is returned as the body of an attestation token from
     the attestation service. It can be used to ensure that the attestation service
     received the policy object sent from the client without alteration.
-    
-    :param policy_resolution: The result of the policy set or
-        reset call.
+
+    :param policy_resolution: The result of the policy set or reset call.
     :type policy_resolution: azure.security.attestation.PolicyModification
     :param policy_signer: If the call to `set_policy` or `reset_policy`
-        had a :class:`AttestationSigningKey` parameter, this will be the certificate
+        had a `signing_certificate` parameter, this will be the certificate
         which was specified in this parameter.
     :type policy_signer: azure.security.attestation.AttestationSigner
     :param str policy_token_hash: The hash of the complete JSON Web Signature
         presented to the `set_policy` or `reset_policy` API.
 
     """
+
     def __init__(self, policy_resolution, policy_signer, policy_token_hash):
-        #type:(PolicyModification, JSONWebKey, str) -> None
+        # type: (PolicyModification, JSONWebKey, str, str) -> None
```

Comment:
you have four positional type hints but only three args here.

Reply:
So weird that VS didn't catch that - it caught a ton of other similar parameter mismatches.